### PR TITLE
Upgrade QSyncthingTray to v0.3.0

### PR DESCRIPTION
**Pull request by kirichkov**
Updates QSyncthingTray to the latest available version - 0.3.0 from 0.2.7

### CODE REVIEW

File: Casks/qsyncthingtray.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'qsyncthingtray' do
-  version '0.2.7'
-  sha256 'b006a0d723560a953e27f5096084d1a57602241e1b60679892b0e65bed261dba'
+  version '0.3.0'
+  sha256 'a63be2b48c6dc1d4c9cf72f9f720fb57bd49f6a6ef507ab6ad56c125876ff80b'
 
-  url "https://github.com/sieren/QSyncthingTray/releases/download/#{version}/QSyncthingTray_OSX.dmg"
+  url "https://github.com/sieren/QSyncthingTray/releases/download/#{version}/QSyncthingTray_0.3_MAC.dmg"
```
Comment (vitorgalvao) [inline Casks/qsyncthingtray.rb]: Interpolation should also be used in the end.

File: Casks/qsyncthingtray.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
   appcast 'https://github.com/sieren/QSyncthingTray/releases.atom',
           :sha256 => '2ca0ae9d1c907c6e86e05f7154e137236febd8c4d96cc1a6989000509ed147f0'
```
Comment (vitorgalvao) [inline Casks/qsyncthingtray.rb]: Updating this value is now desired. You can typically find it with `curl --silent "{{appcast_url_here}}" | shasum --algorithm 256 | awk '{ print $1 }'`.

Alternatively, you may want to use [`cask-repair`](https://github.com/vitorgalvao/tiny-scripts/blob/master/cask-repair), that can at once update the version of cask (according to what you give it) taking care of updating both the cask’s `sha256` and the appcast’s one, while still giving you the chance to perform other manual fixes to the cask. You can easily install it with `brew install vitorgalvao/tinyscripts/caskrepair`. `cask-repair --help` will show you some options, but as a whole the script will guide you through the update process.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in e9b11545c648f1300e7fca5fcb6e89277a370fc3. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by kirichkov** [comment]
Thanks for your help!

**Pull request closed without merging**